tests/secrets/test_local_filesystem.py

```diff
@@ -220,24 +223,30 @@ def test_missing_file(self, mock_exists):
                    a: b
                  extra__google_cloud_platform__keyfile_path: asaa""",
                 {
-                    "conn_a": "mysql://hosta",
-                    "conn_b": ''.join(
-                        """scheme://Login:None@host:1234/lschema?__extra__=%7B
-                        %22extra__google_cloud_platform__keyfile_dict%22%3A+%7B%22a%22%3A+%22b%22%7D%2C
-                        +%22extra__google_cloud_platform__keyfile_path%22%3A+%22asaa%22%7D""".split()
-                    ),
+                    "conn_a": {'conn_type': 'mysql', 'host': 'hosta'},
+                    "conn_b": {
+                        'conn_type': 'scheme',
+                        'host': 'host',
+                        'schema': 'lschema',
+                        'login': 'Login',
+                        'password': 'None',
+                        'port': 1234,
+                        'extra_dejson': {
+                            'extra__google_cloud_platform__keyfile_dict': {'a': 'b'},
```

Comment:
Here we should have json as a string.

Reply:
then you need to change your yaml file like this:
```
               extra_dejson:
                 extra__google_cloud_platform__keyfile_dict: '{"a": "b"}'
                 extra__google_cloud_platform__keyfile_path: asaa""",
                {
                    "conn_a": {'conn_type': 'mysql', 'host': 'hosta'},
                    "conn_b": {
                        'conn_type': 'scheme',
                        'host': 'host',
                        'schema': 'lschema',
                        'login': 'Login',
                        'password': 'None',
                        'port': 1234,
                        'extra_dejson': {
                            'extra__google_cloud_platform__keyfile_dict': '{"a": "b"}',
                            'extra__google_cloud_platform__keyfile_path': 'asaa',
                        },
                    },
                },
```